app/views/organisations/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with(model: organisation) do |form| %>
+  <% if organisation.errors.any? %>
```

Comment:
This is the section that is displayed on error and is as generated by the scaffold generator. I'll strip it out into a generic error partial soon as that makes it easier make error messages consistent.